content/hi/docs/reference/glossary/kubectl.md

```diff
@@ -0,0 +1,18 @@
+---
+title: kubectl
```

Comment:
The **_title_** field needs to have localized word for `kubectl` for eg: `क्यूबसीटीएल (Kubectl)`

Note: The [CHANGELOG-1.9](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.9.md#cli-1 )  states the following:

> The canonical pronunciation of kubectl is "cube control". 

The preferred option to have the localised version could also be `क्यूबकण्ट्रोल (Kubectl)` or `क्यूबकटल (Kubectl)` . There is very less consensus with how to pronounce Kubectl.

I will let the `language/hi` reviewers/owners make the final decision and confirmation on this.

Reply:
Hindi being a very phoentic language, makes this a tough decision but since the canonical pronunciation is "cube control", I think we can go with `क्यूबकण्ट्रोल (Kubectl)`.

Reply:
I think it is better to with `Kubectl` as the title, as other localizations have also done the same. WDYT?